extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/WindowOperatorQueryFrameProcessor.java

```diff
@@ -177,12 +177,12 @@ public ReturnOrAwait<Object> runIncrementally(IntSet readableInputs)
      *
      *  Future thoughts: {@link https://github.com/apache/druid/issues/16126}
      *
-     *  1. We are writing 1 partition to each frame in this way. In case of low cardinality data
+     *  1. We are writing 1 partition to each frame in this way. In case of high cardinality data
      *      we will me making a large number of small frames. We can have a check to keep size of frame to a value
```

Comment:
nit : can correct the typo here as well `we will be`

Reply:
Done